app/assets/stylesheets/styles/components/guides/ui/_actions.css.scss

```diff
@@ -20,6 +20,15 @@
       color: lighten($of-green, 20%);
     }
 
+    i.fa-arrows-v {
+      line-height: 1.6;
+      opacity: 0;
+    }
+
+    li:hover > i.fa-arrows-v {
```

Comment:
Merge rule `li:hover > i.fa-arrows-v` with rule on line 18<br>Selector should have depth of applicability no greater than 3, but was 4